config/apin_O3_agg.yml

```diff
@@ -64,5 +64,35 @@ model_configurations:
       decay: 0.00001
       verbose:  1
       classifier: False
-
+    gru:
```

Comment:
I just realized that these configs will break the original train and run scripts, as those scripts will run through every model configuration / model in the config and train / run it, however those scripts are not yet configured to run your GRU (as you've made a separate script to run them). The simplest solution may be to make a separate config for the GRU, or to pull the  'GRU' portion of the config into its own un-nested dictionary. 

Also, I'm planning making a Github Actions CI script to catch some of these things - hopefully within the next few days.

Reply:
Yeah I did the lazy option and just split the config files. I believe all subsequent dependencies in the RNN and SHAP stuff should be updated to reflect the changes.